src/main/routes/cookies.ts

```diff
@@ -0,0 +1,9 @@
+import * as express from 'express'
+
+import { Paths as AppPaths } from 'app/paths'
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(AppPaths.cookiesPage.uri, function (req, res) {
```

Comment:
These should be behind feature toggle as well.

Another things is that "fat arrow" syntax is preferred. Please also add types.